go.mod

```diff
@@ -204,5 +204,5 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.4.0-sdk-v0.46.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint v0.34.20 => github.com/celestiaorg/celestia-core v1.8.0-tm-v0.34.20
+	github.com/tendermint/tendermint v0.34.20 => github.com/sweexordious/celestia-core v0.34.14-celestia.0.20221201135919-1692b16b341e
```

Comment:
just triple checking that this is the only blocker here?

Reply:
Yes, once the core PR is merged, we can start merging everything